examples/produce_v3/javascript/main.js

```diff
@@ -0,0 +1,52 @@
+// if the endpoint is https, please use https package
+const http = require('http');
+
+// handling streaming response
+var recordReceiptCounter = 1;
+callback = function(response) {
+    response.on(
+        'data',
+        function (data) {
+            let chunk = data.toString('utf8');
+            if (chunk.trim()) {
+                console.log("Receipt for record #" + recordReceiptCounter + " is");
+                console.log(JSON.parse(chunk));
+                recordReceiptCounter += 1;
+            }
+        });
+
+    response.on('end', function () {
+        console.log("Stream is closed!");
+    });
+}
+
+async function main() {
+    // please update the below information to fit your Kafka Rest endpoint
+    const host = 'localhost';
+    const port = '8082';
+    const clusterId = 'ZXBWzl8VQ5WTB_hgEAuGeQ';
+    const topic = 'topic_1';
+    let options = {
```

Comment:
Not sure about the `options` and the `request` but if it can be changed to `const`, will be better to have it immutable as possible